lib/ansible/modules/packaging/os/pkg5.py

```diff
@@ -152,11 +152,13 @@ def ensure(module, state, packages, params):
                 '-q', '--'
             ] + to_modify
         )
-        response['rc'] = rc
+        response['pkg_rc'] = rc
```

Comment:
instead of renaming you can retun failed=False and 'rc' will not be considered for the failure condition

Reply:
Ah, I see this behaviour changed recently, in fceb7112.  Good news.